### DISCUSSION TITLE
A320 ESC Pause issue ?

### DISCUSSION THREAD

**Discussion by fabrcro** [flybywiresim/aircraft#4895] [Q&A] [answered]
Hello
to me happens with both A320: Microsoft and FlyByWire - if you pause with the "ESC" (not the dinamic pause) when you back you find THR to TOGA, Flap 2 and Spoilers to 50% - very annoying if you were flying

Video: [https://youtu.be/HedNrkBBDU0] 

Same to you ?

rgds

**Comment by CptBayer** [comment]
yes ... same to alot of us ... guess they are working on it ... The more strange thing is, that the pauseis no real pause !!! Fuel consumption is still alive and the clock is also running forward ... hmmm ... 

greets Klaus

**Reply by fabrcro** [reply]
This evening the behaviour is different with last development version of FlyByWire, no more TOGA but it disengages A/THR and as usual Flap 2 and spoilers 50%... curious.

**Comment by hotshotp** [comment, marked as answer]
this is not an a32nx issue but a sim issue which will be fixed by asobo

**Reply by ExampleWasTaken** [reply]
![image](https://user-images.githubusercontent.com/58574351/119423624-f6e9c880-bd03-11eb-9b94-0ad159049cd1.png)

**Reply by hotshotp** [reply]
LOL I mean I know it's the answer to the issue so I just mark it as the answer 😆

**Reply by ExampleWasTaken** [reply]
> 
> 
> LOL I mean I know it's the answer to the issue so I just mark it as the answer 😆

Sure! I just thought of that meme and couldn't resist 🤣
